thavalon-webapp/src/components/Login.tsx

```diff
@@ -39,8 +42,8 @@ export function Login(props: LoginProps) {
      * @param event The event caused by submission.
      */
     async function OnSubmit(data: LoginData) {

```

Comment:
Can we make this camelCase to distinguish it from a React component since it's passed around in the render function?